beam_nuggets/io/relational_db_api.py

```diff
@@ -358,7 +361,6 @@ def write_record(self, session, create_insert_f, record_dict):
                 record=record_dict
             )
             session.execute(insert_stmt)
```

Comment:
Not sure about SQLAlchemy behavior here. If the behavior is sending each stmt directly the DB, then another potential improvement could be to create a single insert stmt for the whole batch using, e.g., something like [here](https://docs.sqlalchemy.org/en/14/orm/session_api.html#sqlalchemy.orm.Session.execute.params.params) or [here](https://docs.sqlalchemy.org/en/14/core/dml.html?highlight=expression%20insert#sqlalchemy.sql.expression.Insert.values.params.*args)

Reply:
@mohaseeb Correct. I ended up creating my own batch insert mod based on your project and the only thing I changed was around removing assumptions that `record` is a dict. SQLAlchemy while generating insert statement supports both a single row (dict ) or multiple rows (list of dicts).